src/parser.cc

```diff
@@ -247,7 +247,7 @@ static void insertIncludedElement(sdf::SDFPtr _includeSDF,
 
   ElementPtr proxyModelFrame = _parent->AddElement("frame");
   const std::string proxyModelFrameName =
-      "merged__" + model->Name() + "__model__";
+      "_merged__" + model->Name() + "__model__";
```

Comment:
Just double checking, single underscore as a prefix, not two underscores, right?

Reply:
Yes, just single. Names that start and end with a double underscore are not permitted per http://sdformat.org/tutorials?tut=pose_frame_semantics&cat=specification&#unique-names-and-reserved-names, but we still wanted this name to have a special meaning.

Reply:
I see, thanks for the clarification!